homeassistant/components/tessie/strings.json

```diff
@@ -301,6 +301,24 @@
           "high": "[%key:component::tessie::entity::select::climate_state_seat_heater_left::state::high%]"
         }
       },
+      "climate_state_seat_fan_front_left": {
+        "name": "Seat cooler left",
+        "state": {
+          "off": "[%key:common::state::off%]",
+          "low": "Low",
```

Comment:
You should probably reuse the reference from seat heater since I don't think there needs to be a new translation for Low, Medium, High

Reply:
Thanks for the quick review.  I've updated the strings.json per your advice.